chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -1422,7 +1422,7 @@ mcCombinations.prototype.retrieveCredentialsCallback = function (credentials) {
 			if (a.deep == b.deep) return 0
 		})
 		
-		if (buttons.length > 0) return buttons[0]
+		if (buttons.length > 0 && buttons[0].deep < 150) return buttons[0]
```

Comment:
are you sure that's a good idea? what happens if that fails? no submit event?

Reply:
@limpkin we will fallback to form submit event if it fails

Reply:
i think it's better than triggering click event on the wrong element, users report this kind of issue quite often

Reply:
good point